docs/sdk/web3-react/guides/01-connect-wallet.md

```diff
@@ -1,4 +1,165 @@
+
+```typescript reference title="Importing Connector dependencies" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/856dbb002e7f38120554ef226f4309c96ce6ea79/web3-react/src/libs/injected.ts#L1-L2
+```
+
+We then utilize the templated `initializeConnector` function with a type of `MetaMask`:
+
+```typescript reference title="Initializing the MetaMask connector" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/81ec93e97b0afded621e177fe5f34fc9f98f80b0/web3-react/src/libs/injected.ts#L7-L9
+```
+
+By passing in `MetaMask` as the templated parameter, we define the function's required input parameters. In this case, the only parameter we need to pass is an instance of `Metamask`, which receives the `actions` and `onError` parameters. The initial parameter defines the actions that `web3-react` can execute on a local store it creates for the connector, while the latter parameter defines the connector's behavior when an error in encountered.
+
+The return type of the function is a tuple of the initialized **connector** and the **hooks** that we can use on it. Using this tuple, we create an instance of a [`Connection`](https://github.com/Uniswap/examples/blob/856dbb002e7f38120554ef226f4309c96ce6ea79/web3-react/src/libs/connections.ts#L10) type, by setting the `type` attribute to `INJECTED`:
+
+
+```typescript reference title="Creating a connection instance" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/856dbb002e7f38120554ef226f4309c96ce6ea79/web3-react/src/libs/injected.ts#L16-L20
+```
+
+Finally, we return the instance we created, which is added to the list of prioritized connectors. 
+
+:::info
+For help on creating the rest of the supported connectors of this examples, please visit our [connectors](./connectors.md) page!
+:::
+
+
+## Connecting and  disconnecting the application to the connector
+
+Having built our Injected Connector, we can now build the component that gives our application the ability to communicate with that connector. The basic functionality we would like to support is **connecting** and **disconnecting**  the connector to our application. To achieve this, we create a reusable [Option component](https://github.com/Uniswap/examples/blob/main/web3-react/src/libs/components/Option.tsx) that supports rendering the UI and managing state for any connector that we want to support in our application:
+
+```typescript reference title="Creating the Option component" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/81ec93e97b0afded621e177fe5f34fc9f98f80b0/web3-react/src/libs/components/Option.tsx#L5-L11
+```
+
+The component receives 5 parameters:
```

Comment:
5 named parameters

These are named, so you should refer to their names below when explaining them